src/components/AppComponents/AppHeader/AppHeader.tsx

```diff
@@ -35,75 +35,79 @@ const AppHeader: FC<HeaderProps> = () => {
   const selectedArea = useSelectedArea();
 
   return (
-    <StyledHeader
-      $background="white"
-      as="header"
-      $justifyContent={["space-between"]}
-      $alignItems={["center"]}
-      $zIndex="fixedHeader"
-      $position={"relative"}
-    >
-      <OakFlex
-        $justifyContent={"space-between"}
-        $flexGrow={1}
-        $alignItems={"center"}
+    <header>
```

Comment:
Moving the `<header>` so that it wraps `StyledHeader` and `AppHeaderMenu` so that the page outline is not changed by me moving things around to get the stacking order I need.